tests/integration/integration_test.go

```diff
@@ -449,7 +396,7 @@ status: {}
 					"--create-pr",
 					"--branch",
 					branch,
-					"nginx-catalog/weaveworks-nginx")
+					"nginx-catalog/weaveworks-nginx/v0.1.0")
```

Comment:
We should also have one without version right? Because that also should work. Or do we have one I just don't see it? :D

Reply:
I think it didn't work actually, which is why I modified it!